### PR TITLE
Added CONTRIBUTING.md file to repository

### LINKED ISSUE #8: Create a CONTRIBUTING.md file
This repository needs a CONTRIBUTING.md file so we can have a consistent structure, along with outlines for how to get started on creating a new scenario due to Katacoda's sparse documentation.

### PR DESCRIPTION
**Pull request by mdodell**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- [Troubleshooting Steps](#troubleshooting-steps)
+- [Contribution Workflow](#contribution-workflow)
+
+## Prerequisites
+The Katacoda CLI is recommended in order to create a new scenario. You can install the Katacoda CLI by running `npm install katacoda-cli --global` in the terminal.
+
+## Katacoda Documentation
+A link to the Katacoda documentation can be found [here](https://www.katacoda.community/welcome.html).
+
+Additionally, it is recommended that you look at the other scenarios in the `cyberark/katacoda-scenarios` to see how Katacoda has been used in other tutorials.
```
Comment (sgnn7) [inline CONTRIBUTING.md]: This isn't a full http link

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [Katacoda Documentation](#katacoda-documentation)
+- [Building and Testing](#building-and-testing)
+- [Troubleshooting Steps](#troubleshooting-steps)
+- [Contribution Workflow](#contribution-workflow)
+
+## Prerequisites
+The Katacoda CLI is recommended in order to create a new scenario. You can install the Katacoda CLI by running `npm install katacoda-cli --global` in the terminal.
```
Comment (sgnn7) [inline CONTRIBUTING.md]: Apply word wrap to the whole file (80-100 chars)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Katacoda Documentation
+A link to the Katacoda documentation can be found [here](https://www.katacoda.community/welcome.html).
+
+Additionally, it is recommended that you look at the other scenarios in the `cyberark/katacoda-scenarios` to see how Katacoda has been used in other tutorials.
+
+## Building and Testing
+The Katacoda scenarios can be tested using the `katacoda-cli`. In order to validate that the structure of your scenario is properly defined, you can run `katacoda validate:all .` in the root directory of the `cyberark/katacoda-scenarios` directory.
```
Comment (sgnn7) [inline CONTRIBUTING.md]: Link to repo needs to be the full link

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+In order for your scenario to be properly validated, all extraneous files (if any) must be included directly in the `assets` directory of your scenario.
+
+Our repository also runs this command automatically using a Github action.
+
+## Troubleshooting Steps
+Katacoda may take some time to properly load in changes to your scenario. It can take anywhere from 5 to 30 minutes for the scenario to propagate onto Katacoda.
+
+If it takes longer than this to run, we highly recommend checking your Github log and pushing again if necessary.
```
Comment (sgnn7) [inline CONTRIBUTING.md]: Would be good to add info on how to know when it's done

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+## Contribution Workflow
+1. Fork the repository to your Github account (this is necessary in order to set up a [Katacoda webhook](https://katacoda.com/teach/git-hosted-scenarios)
+2. If issue for relevant change has not been created, open one [here](https://github.com/cyberark/katacoda-scenarios/issues)
+3. Add the `implementing` label to the issue once you begin to work on it
+4. Commit your changes (`git commit -m 'Add X feature that does Y'`)
```
Comment (sgnn7) [inline CONTRIBUTING.md]: Using `-m` in this way omits the commit description - we shouldn't use in this manner. You can probably just leave `git commit` by itself here.

**Review by sgnn7 — CHANGES_REQUESTED** [review]
@mdodell Just a few small comments

File: CONTRIBUTING.md (hunk 4 above)
Comment (mdodell) [reply CONTRIBUTING.md]: Unfortunately, the only way is to refresh your changes and wait for them to populate. If it's not done after 30 minutes, there was probably an issue.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mdodell) [reply CONTRIBUTING.md]: Are you referring to the Katacoda documentation link or the repository path?

File: CONTRIBUTING.md (hunk 3 above)
Comment (mdodell) [reply CONTRIBUTING.md]: Is the standard to link the repository anytime you are referring to it?

**Review by sgnn7 — APPROVED** [review]

**Review by sgnn7 — CHANGES_REQUESTED** [review]
Oops - didn't see that commit message needs fixing

**Review by sgnn7 — APPROVED** [review]

**Pull request merged**